backend/inspirehep/migrator/tasks.py

```diff
@@ -291,14 +291,16 @@ def create_records_from_mirror_recids(self, recids):
             raise
         except Exception:
             LOGGER.exception("Cannot migrate record", recid=recid)
-            continue
+            raise
     try:
         db.session.commit()
     except Exception:
```

Comment:
we re-raise for any `Exception`, but we don't retry for `Exception`. We probably should catch and re-raise separately the errors that actually occur on `commit` and retry those, keeping the generic `Exception` catching to avoid the chord blowing up on unexpected exceptions.

Reply:
:+1: